src/commands/kick.ts

```diff
@@ -16,7 +16,7 @@ async function kickHandler(interaction: ChatInputCommandInteraction): Promise<vo
 	const users = interaction.options.getString('users', true);
 	const reason = interaction.options.getString('reason', true);
 
-	const userIds = [...new Set(Array.from(users.matchAll(new RegExp(MessageMentions.UsersPattern, 'g')), match => match[1]))];
+	const userIds = [...new Set(Array.from(users.matchAll(new RegExp(/\d{17,18}/, 'g')), match => match[0]))];
```

Comment:
Since you're using a regex literal you can remove the stuff to add the global flag:
```ts
users.matchAll(/\d{17,18}/g, ...)
```